### DISCUSSION TITLE
Dashboard provisioning: Enable adding a tag to provisioned dashboards

### DISCUSSION THREAD

**Discussion by StianOvrevage** [grafana/grafana#54217] [Dashboards, Panels & Visualizations]
# Feature requested / solution

Add an optional

    addTags:
      - provisioned

field to the dashboard provisioning configuration ( https://grafana.com/docs/grafana/latest/administration/provisioning/#dashboards ).

Guessing this has to be done in https://github.com/grafana/grafana/blob/main/pkg/services/provisioning/dashboards/file_reader.go but I'm not proficient enough with the Grafana codebase to pitch a PR.

# Problem

It's not possible to know which dashboards are actually editable before doing changes and trying to save and getting the `This dashboard cannot be saved from the Grafana UI because it has been provisioned from another source.` message.

# Background

We have an environment with a mix of provisioned dashboards (mainly from https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14 but also other and internal sources).

For provisioned dashboards we have `allowUiUpdates: false` to avoid drift. ( https://grafana.com/docs/grafana/latest/administration/provisioning/#dashboards ).

Many of the dashboards are `editable` upstream ( https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/grafana-overview.yaml#L45 ) making it seem like the dashboard is editable when in practice it is not.

# Things I've tried

- Looking into the sidecar, but that's just a basic ConfigMap synchronizer that has no knowledge of Grafana
- Add tags to the dashboards upstream, but they are hardcoded ( https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/controller-manager.yaml#L1081 ).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
